benchmark/build_csv.py

```diff
@@ -1,8 +1,14 @@
+import argparse
```

Comment:
Let's maybe integrate this in main cli ?

Reply:
Thanks, done, looks indeed better like that. Perharps we should do the same for `clip_benchmark_export_wds` (in another PR).